articles/quickstart/webapp/nextjs/01-login.md

```diff
@@ -164,6 +166,28 @@ The `user` property contains sensitive information and artifacts related to the
 - Ensure that the SDK has loaded successfully by checking that no `error` was produced.
 - Check the `user` property to ensure that Auth0 has authenticated the user before React renders any component that consumes it.
 
+### From a Client Component
+
+The profile information is available through the `user` property exposed by the `getSession` function. Take this [Server Component](https://nextjs.org/docs/getting-started/react-essentials#server-components) as an example of how to use it:
+
+```jsx
+import { getSession } from '@auth0/nextjs-auth0';
+
+export default async function ProfileServer() {
+  const { user } = await getSession();
+
+  return (
+      user && (
+          <div>
+            <img src={user.picture} alt={user.name} />
+            <h2>{user.name}</h2>
+            <p>{user.email}</p>
+          </div>
+      )
+  );
+}
+```
+
```

Comment:
Should we include a panel with a link to https://github.com/auth0/nextjs-auth0#using-this-sdk-with-react-server-components?

Reply:
I don't think that's necessary and complicates the quickstart - they'll get the warning in the console.